common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/domain/metrics/ExposureRiskMetadata.java

```diff
@@ -19,23 +19,46 @@
   @Range(min = MIN_RISK_LEVEL, max = MAX_RISK_LEVEL,
       message = "Risk Level must be in between " + MIN_RISK_LEVEL + " and " + MAX_RISK_LEVEL + ".")
   private final Integer riskLevel;
+  /**
+   * The risk level reported by the client from check-in-based presence tracing.(0 to 3).
+   */
+  @Range(min = MIN_RISK_LEVEL, max = MAX_RISK_LEVEL,
+      message = "Risk Level must be in between " + MIN_RISK_LEVEL + " and " + MAX_RISK_LEVEL + ".")
+  private final Integer ptRiskLevel;
```

Comment:
Please consider embedding the PT* values in its own entity and just inline the attributes into the ExposureRiskMetadata. This way:
- we have concentrated pt logic in an entity and in case of changes it is easier to adjust
- the exposure risk meta data class doesn't blow up so much.

Since it is explicitly mentioned for other entities to be inline (f.e. UserMetadataDetails) we should clarify this